### PR TITLE
AG-9212 - Fix theme examples.

### PR DESCRIPTION
**Pull request by alantreadway**
https://ag-grid.atlassian.net/browse/AG-9212

Theme example improvements:
- Fix use of `cartesian` and `polar` properties that are no longer present.
- Move data into `data.ts` files to declutter examples.
- Rework a provided example to improve maintainability.

General improvements:
- Remove redundant use of `autoSize: true`.

### CODE REVIEW

File: packages/ag-charts-website/src/content/docs/themes/_examples/advanced-theme-polar/main.ts [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+  overrides: {
+    common: {
+      title: {
+        fontSize: 24,
+      },
+      padding: {
+        left: 70,
+        right: 70,
+      },
+      axes: {
```
Comment (manapeirov) [inline packages/ag-charts-website/src/content/docs/themes/_examples/advanced-theme-polar/main.ts]: does the polar chart need this theme config? it doesn't have axes

File: packages/ag-charts-website/src/content/docs/themes/_examples/advanced-theme-polar/main.ts [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+          line: {
+            color: "gray",
+          },
+          tick: {
+            color: "gray",
+          },
+        },
+      },
+    },
+    line: {
```
Comment (manapeirov) [inline packages/ag-charts-website/src/content/docs/themes/_examples/advanced-theme-polar/main.ts]: This stuff can be removed from the theme for polar

File: packages/ag-charts-website/src/content/docs/themes/_examples/stock-themes/main.ts [hunk 3]
```diff
@@ -1,24 +1,13 @@
 import { AgChartOptions, AgChartTheme, AgChart } from 'ag-charts-community'
+import { getData } from './data';
 
 const options: AgChartOptions = {
   container: document.getElementById('myChart'),
   theme: 'ag-default-dark',
-  autoSize: true,
```
Comment (manapeirov) [inline packages/ag-charts-website/src/content/docs/themes/_examples/stock-themes/main.ts:6]: Did you mean to delete `autoSize` here?

File: packages/ag-charts-website/src/content/docs/themes/_examples/advanced-theme-polar/main.ts (hunk 1 above)
Comment (alantreadway) [reply packages/ag-charts-website/src/content/docs/themes/_examples/advanced-theme-polar/main.ts]: The intent is to show the same theme config can be used in both cases.

File: packages/ag-charts-website/src/content/docs/themes/_examples/advanced-theme-polar/main.ts (hunk 2 above)
Comment (alantreadway) [reply packages/ag-charts-website/src/content/docs/themes/_examples/advanced-theme-polar/main.ts]: The intent is to show the same theme config can be used in both cases.

File: packages/ag-charts-website/src/content/docs/themes/_examples/stock-themes/main.ts (hunk 3 above)
Comment (alantreadway) [reply packages/ag-charts-website/src/content/docs/themes/_examples/stock-themes/main.ts:6]: Yes, it's the default so shouldn't be necessary?

Comment (alantreadway) [reply packages/ag-charts-website/src/content/docs/themes/_examples/stock-themes/main.ts:6]: I've removed all redundant `autoSize: true` from examples to cleanup generally.

File: packages/ag-charts-website/src/content/docs/themes/_examples/advanced-theme-polar/main.ts (hunk 1 above)
Comment (alantreadway) [reply packages/ag-charts-website/src/content/docs/themes/_examples/advanced-theme-polar/main.ts]: Collapsed the two examples back to one.

File: packages/ag-charts-website/src/content/docs/themes/_examples/advanced-theme-polar/main.ts (hunk 2 above)
Comment (alantreadway) [reply packages/ag-charts-website/src/content/docs/themes/_examples/advanced-theme-polar/main.ts]: Collapsed the two examples back to one.

**Comment by alantreadway** [comment]
@manapeirov feedback addressed, PTAL - thanks!

**Review by manapeirov — APPROVED** [review]
LGTM thank you!

**Pull request merged**